internal/grpc/services/gateway/storageprovider.go

```diff
@@ -269,6 +269,7 @@ func (s *svc) UnsetArbitraryMetadata(ctx context.Context, req *storageproviderv0
 }
 
 func (s *svc) Stat(ctx context.Context, req *storageproviderv0alphapb.StatRequest) (*storageproviderv0alphapb.StatResponse, error) {
+	// TODO(refs) do we need to append home to every stat request?
```

Comment:
Not at all, the gateway doesn't do any path manipulation, only find the available storage providers and send request there, is the storage provider that exposes /home. 

Gateway is just dummy :)